site/docs/pages/identity/introduction.mdx

```diff
@@ -13,17 +13,40 @@ In order to use **Identity Kit** you must also install `graphql@14 graphql-reque
 :::code-group
 
 ```bash [npm]
-npm install @coinbase/onchainkit graphql@14 graphql-request@6
+npm install @coinbase/onchainkit graphql@14 graphql-request@6 @tanstack/react-query@5.x
 ```
 
 ```bash [yarn]
-yarn add @coinbase/onchainkit graphql@14 graphql-request@6
+yarn add @coinbase/onchainkit graphql@14 graphql-request@6 @tanstack/react-query@5.x
 ```
 
 ```bash [pnpm]
-pnpm add @coinbase/onchainkit graphql@14 graphql-request@6
+pnpm add @coinbase/onchainkit graphql@14 graphql-request@6 @tanstack/react-query@5.x
 ```
 
+## React
+
+If you are using any of the provided react hooks, you will need to install and configure `@tanstack/react-query` and wrap your app in `<QueryClientProvider>`.
```

Comment:
Where is the doc for `QueryClientProvider`? I feel like have a link that goes to their main doc, can be useful.